QHotkey/qhotkey_x11.cpp

```diff
@@ -1,7 +1,19 @@
 #include "qhotkey.h"
 #include "qhotkey_p.h"
-#include <QDebug>
-#include <QX11Info>
+
+#if QT_VERSION >= QT_VERSION_CHECK(6, 2, 0)
+	#include <QGuiApplication>
+	
+	#define _NATIVE_EVENT_RESULT qintptr
+#elif QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+	#error "Qt 6.2.0 or greater is required when using Qt6"
```

Comment:
Do we need to handle this check here? We already a have check in CMake / QMake. I would just check for major version just to simplify the logic.

Reply:
Not really :)

I putted it for the sake of completeness of the C++ code, so it can be independent of the build system.

I will remove this part then.

Reply:
Modifications pushed. I just removed those lines.

Please reopen if it's not ok.